content/docs/components/divider/theming.mdx

```diff
@@ -3,4 +3,173 @@ id: divider
 scope: theming
 ---
 
-TODO
+The `Divider` component is a single part component. All of the styling is applied
+directly to the `chakra.hr` element.
+
+> To learn more about styling single part components, visit the [Component Style](/docs/styled-system/component-style#styling-single-part-components) page
+
+## Theming properties
+
+The properties that affect the theming of the `Divider` component are:
+
+- `variant`: The visual variant of the divider. Defaults to `solid`.
+- `colorScheme`: The color scheme of the divider.
+- `size`: The size of the divider.
+
+## Theming utilities
+
+- `defineStyle`: a function used to create style objects.
+- `defineStyleConfig`: a function used to define the style configuration for a single part component.
+
+```js
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
+```
+
+## Customizing the default theme
+
+```js
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
+
+const thicc = defineStyle({
+    border: '5px solid', // change the appearance of the border
```

Comment:
This will not work. You should break down the `border` property into `borderWidth` and `borderStyle`. That's why you can't see the border in your sandbox. Do this in all of the cases.